src/main/java/org/humancellatlas/ingest/core/MetadataDocumentMessageBuilder.java

```diff
@@ -33,7 +30,7 @@
     private ValidationState validationState;
     private int assayIndex;
     private int totalAssays;
-    private Collection<String> envelopeIds;
+    //private Collection<String> envelopeIds;
```

Comment:
Rather than commenting out redundant code, let's just remove it. Makes comprehension and maintenance easier in the future. We have the history of the file in the repository if we need to look up an old implementation.